reconcile/test/ocm/oum/test_oum_base.py

```diff
@@ -340,9 +349,12 @@ def test_build_spec_from_config_cluster_admin_without_capability(
         },
     )
     assert len(specs) == 1
-    assert specs[0].roles == {}
-    assert specs[0].cluster == cluster
-    assert len(specs[0].errors) == 1
+    if errors:
```

Comment:
use parametrize tests and conditional assert together seems not serving the purpose of parametrize well, I suggest to extract more assertion into params then make test body consistent. For example, to have `expected_roles` in params, then

```python
assert len(specs[0].errors) == errors
assert specs[0].roles == expected_roles
```